Distribution/Server/Features/Users.hs

```diff
@@ -412,6 +421,11 @@ userFeature templates usersState adminsState
           overrideResponse <- msum <$> runHook authFailHook err
           throwError (fromMaybe defaultResponse overrideResponse)
 
+    -- Check if there is an authenticated userid, and return info, if so.
+    getAuthentication :: ServerPartE (Maybe (UserId, UserInfo))
```

Comment:
This should be called `checkAuthenticated` since that's what it wraps and that's the pattern here.